daphne_worker/src/config.rs

```diff
@@ -84,6 +84,18 @@ pub(crate) struct DaphneWorkerConfig<D> {
 
     /// Indicates if DaphneWorker is used as the Leader.
     is_leader: bool,
+
```

Comment:
Sugestion: Instead of having each of these be individually optional, It might make more sense to group them into one struct and make that struct optional:
```rust
pub(crate) taskprov_config: Option<TaskprovConfig>,
```

That way you don't have to deal with edge cases where the hpke_collector_config is present but the vdaf_verify_key_init is missing.

Reply:
good idea